client/helpers.go

```diff
@@ -63,12 +63,12 @@ var (
 	supportedServiceRegion     *SupportedServiceRegionsData
 )
 
-var notFoundErrorPrefixes = []string{
-	"ResourceNotFoundException",
-	"WAFNonexistentItemException",
+var notFoundErrorSubstrings = []string{
+	"InvalidRequestException",
```

Comment:
Good question we had a report of an error:

```
athena.work_groups: failed to resolve table "aws_athena_work_group_query_executions": error at github.com/cloudquery/cq-provider-aws/resources/services/athena.fetchAthenaWorkGroupQueryExecutions[work_groups.go:526] operation error Athena: GetQueryExecution, https response error StatusCode: 400, RequestID:<REDACTED-UUID>, InvalidRequestException: QueryExecution <REDACTED-UUID> was not found
```

Maybe instead of ignoring it for all resource we just add it to that single resource?